crates/release_plz/src/main.rs

```diff
@@ -29,6 +29,7 @@ async fn main() -> anyhow::Result<()> {
             let request: ReleaseRequest = cmd_args.into();
             release_plz_core::release(&request).await?;
         }
+        args::Command::ShellCompletions(cmd_args) => cmd_args.print()?,
```

Comment:
I couldn't think of any good names, so I just used `ShellCompletions`. I'm very open to suggestions here.

Reply:
I would like to have "verbs" for commands.
Something like `GenerateCompletions` could work. What do you think? :)

Reply:
> I would like to have "verbs" for commands.

Aha! Good to know.

> Something like GenerateCompletions could work. What do you think? :)

Sounds good to me!
Will do in the next commit patch set / update once I figure out how to update commits here on github.

Reply:
> Will do in the next commit patch set / update once I figure out how to update commits here on github.

just do git push on the branch of your fork. Just like you did before opening the PR :)